src/main/java/ru/yandex/qatools/embed/postgresql/PostgresProcess.java

```diff
@@ -189,6 +189,12 @@ protected void onBeforeProcess(IRuntimeConfig runtimeConfig)
             throws IOException {
         super.onBeforeProcess(runtimeConfig);
         PostgresConfig config = getConfig();
+
+        final File     dbDir   = config.storage().dbDir();
+        if (dbDir.exists() && dbDir.listFiles() != null && dbDir.listFiles().length > 0) {
```

Comment:
I'd prefer to have a conditional `runCmd` here rather than a conditional return. Conditional return would make more sense if we had more than 1 LOC here.